tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/TestHelper.cs

```diff
@@ -88,6 +90,15 @@ public async Task<Process> StartDotnetTestSample(MockTracerAgent agent, string a
                 throw new Exception($"application not found: {sampleAppPath}");
             }
 
+            // Ensure we have the CI Environment variables in the target process
+            foreach (var key in CIEnvironmentValues.Constants.GetConstants())
```

Comment:
IMO `GetConstants` should be defined here, not in DD.Trace

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/5269/commits/c400bda63c9512ec05e6a0a2551dbe6c8e92c123